### PR TITLE
extract_labels command

### PR DESCRIPTION
**Pull request by Br4nd0R**
# Changes
* New extract_labels command extracts all `TranslatedFields` on object in MetaDeploy into JSON format. This will allow an easy way to provide fields that potentially need translating at a later time in a convenient format.

### CODE REVIEW

File: metadeploy/management/commands/extract_labels.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
+import json
+
+from django.apps import apps
+from django.core.management.base import BaseCommand
+
+
+class Command(BaseCommand):
+    help = (
+        "Queries for api_clickthroughagreement records and outputs them in csv format"
```
Comment (davisagli) [inline metadeploy/management/commands/extract_labels.py]:
```suggestion
        "Queries for translatable field values and outputs them in json format"
```

File: metadeploy/management/commands/extract_labels.py [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+            "name": "Name of the site profile",
+            "company_name": "Name of the company",
+            "welcome_text": "Text shown above products on the home page",
+            "copyright_notice": "Copyright notice shown in the footer of the page",
+        },
+    }
+
+    def handle(self, *args, **options):
+        translatable_labels = {}
+        for obj in self.translatable_objects.keys():
```
Comment (davisagli) [inline metadeploy/management/commands/extract_labels.py]:
```suggestion
        for obj in self.translatable_objects:
```

(iterating over a dict actually iterates over its keys implicitly -- if you prefer to be explicit that's fine though)

**Comment by Br4nd0R** [comment]
@davisagli - This should be ready for testing in metadeploy-stg. Feel free to give it a once over when you have time.

File: metadeploy/management/commands/extract_labels.py [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+            "description": "Description of the product",
+            "click_through_agreement": "Users must check a box to agree to this legal text before running the installer",
+            "error_message": "Message regarding what to do if an error occurs during installation",
+        },
+        "PlanTemplate": {
+            "preflight_message": "Message displayed during preflight checks",
+            "post_install_message": "Message displayed after installation completes",
+            "error_message": "Message displayed to users if an error occurs",
+        },
+        # "Plan": {
```
Comment (Br4nd0R) [inline metadeploy/management/commands/extract_labels.py]: I just wanted to leave these in, just incase we wanted to incorporate them at a later time.

**Pull request merged**